extension/parquet/column_writer.cpp

```diff
@@ -785,13 +785,21 @@ struct ParquetCastOperator : public BaseParquetOperator {
 struct ParquetTimestampNSOperator : public BaseParquetOperator {
 	template <class SRC, class TGT>
 	static TGT Operation(SRC input) {
+		auto ts = timestamp_t(input);
```

Comment:
Does parquet support ±infinity? I couldn't find any docs.

Reply:
Doesn't that mean that infinities can't be roundtripped to/from parquet?

Reply:
Think so.